string io_var
bool io_value
int32 timeout_ms # milliseconds
---
# Success if empty
string error
